### PR TITLE
Update the urllib3 version range in salvo/requirements.txt.

### PR DESCRIPTION
**Pull request by gyohuangxin**

### CODE REVIEW

**Comment by gyohuangxin** [comment]
The error message is shown as below:
```
Analyzing: 35 targets (7 packages loaded, 0 targets configured)
Analyzing: 35 targets (7 packages loaded, 0 targets configured)
[0 / 1] checking cached actions
ERROR: /source/salvo/BUILD:5:10: While resolving toolchains for target //:salvo (5c86453): invalid registered toolchain '//toolchains:all': while parsing '//toolchains:all': error loading package '@@rules_java_builtin//toolchains': cannot load '@@bazel_skylib//:bzl_library.bzl': no such file
ERROR: Analysis of target '//:salvo' failed; build aborted
```
@mum4k Do you have any idea on that issue?

**Comment by mum4k** [comment]
That looks like a break introduced via one of our dependencies. I don't have more context on it. If you have time to debug this I would try to either research it online, or reproduce locally and debug.

File: salvo/requirements.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@ requests>=2.25.0
 s3transfer>=0.3.3
 six>=1.15.0
 typed-ast>=1.4.1
-urllib3>=1.26.5
+urllib3>=1.25.4,<1.27
```
[the remark this answers is not in the document]
Comment (gyohuangxin) [reply salvo/requirements.txt]: Sure, you're correct. I've reverted this change.

**Comment by gyohuangxin** [comment]
@mum4k Can you review it again, thanks?

**Review by mum4k — APPROVED** [review]
Nice, thank you for diagnosing this!

**Pull request merged**